code/_onclick/hud/spell_screen_objects.dm

```diff
@@ -214,3 +216,12 @@
 
 	spell.perform(usr)
 	update_charge(1)
+
+/obj/screen/spell/proc/add_channeling()
+	var/image/channel = image(icon = icon, loc = src, icon_state = "channeled", layer = src.layer + 1)
```

Comment:
Making an image like that is negligible, and if you think that is bad look at camerachunk code. Simply horrifying.